### ISSUE TITLE
WaterHeater:HeatPump Field !- Tank Element Control Logic is not exposed

### ISSUE THREAD

**Issue by Ski90Moo** [openstudiocoalition/OpenStudioApplication#773]
Enhancement Request
--------------
This field is exposed in the WaterHeater:HeatPump:WrappedCondenser object, but for some reason it is not exposed on the WaterHeater:HeatPump (EnergyPlus named :PumpedCondenser).

This field is essential for specifying how the water heater heat sources are controlled.

<img width="1280" alt="image" src="https://github.com/user-attachments/assets/cea511ca-1523-4490-8988-eb2fa05aba5a">

## Detailed Description
I tried editing the osm file and adding:

![image](https://github.com/user-attachments/assets/83f6dfa1-0b21-4a00-8b0d-dee35dcf66d1)

But I got the following error:

![image](https://github.com/user-attachments/assets/a95bce73-4d8d-499a-b162-26a1ef189bf9)

So I think there might be something wrong with the SDK as well?

**Comment by macumber** [comment]
Yes, that field seems to be missing from the `OS:WaterHeater:HeatPump` object in the OS SDK https://github.com/NREL/OpenStudio/blob/develop/resources/model/OpenStudio.idd#L31705.  Can you open a new issue for the OS SDK and link it to this one @Ski90Moo?

**Comment by Ski90Moo** [comment]
Done
